src/home_robot_hw/home_robot_hw/env/stretch_abstract_env.py

```diff
@@ -325,6 +453,30 @@ def get_images(self, compute_xyz=False, rotate_images=True):
 
         return imgs
 
+    def get_pose(self, frame, base_frame=None, lookup_time=None, timeout_s=None):
+        """look up a particular frame in base coords"""
```

Comment:
Since the abstract_env will be shared across by users, we should start adding informative docstrings. Specifically, input output and 1 line desc of function between them.